src/renderer/shared/core/index.ts

```diff
@@ -52,14 +52,15 @@ export type { Validator } from './types/validator';
 export { RewardsDestination } from './types/stake';
 export type { Stake, Unlocking } from './types/stake';
 
+export { ProxyType, ProxyVariant } from './types/proxy';
```

Comment:
This import was grouped with other Proxy types, now it's separated
Please move it back